lib/membrane_live/webinars.ex

```diff
@@ -67,37 +48,13 @@ defmodule MembraneLive.Webinars do
           MembraneLive.Webinars.Webinar.t(),
           :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
         ) :: any
-  @doc """
-  Updates a webinar.
-
-  ## Examples
-
-      iex> update_webinar(webinar, %{field: new_value})
-      {:ok, %Webinar{}}
-
-      iex> update_webinar(webinar, %{field: bad_value})
-      {:error, %Ecto.Changeset{}}
```

Comment:
Based on the example below, we can see that output of function will be `{:ok, %Webinar{}} | {:error, %Ecto.Changeset{}}`

Reply:
Right 😅